itwm_example/mco/weighted_mco_model.py

```diff
@@ -0,0 +1,34 @@
+from traits.api import Enum, Bool
+from traitsui.api import View, Item
+
+from force_bdss.api import BaseMCOModel, PositiveInt, WeightedOptimizerEngine
+
+
+class WeightedMCOModel(BaseMCOModel):
+
+    #: Algorithms available to work with
+    algorithms = Enum(
+        *WeightedOptimizerEngine.class_traits()["algorithms"].handler.values
```

Comment:
Out of curiosity, obtaining `Enum` trait values looks messy...

Reply:
It is very hacky indeed. For future reference, this seems to be the most concise way to access the `algorithms` attribute of the `WeightedOptimizerEngine` class without instantiating it, and with no dependency on the traits initialization ordering.